tests/queries/0_stateless/03174_least_greatest_for_null.sql

```diff
@@ -0,0 +1,17 @@
+drop table if exists test1;
+drop table if exists test2;
+
+create table test1(id UInt32, x1 Nullable(Float64), x2 Nullable(Int32)) Engine=MergeTree order by id;
```

Comment:
Instead of generic names `test1` and `test2`, what about `tab` (as base case` and `tab_decimal`?